sdcm/utils/data_validator.py

```diff
@@ -167,7 +167,7 @@ def __init__(self, longevity_self_object, user_profile_name, base_table_partitio
     @property
     def keyspace_name(self):
         if not self._keyspace_name:
-            prepare_write_cmd = self.longevity_self_object.params.get(self.stress_cmds_part, default=[])
+            prepare_write_cmd = self.longevity_self_object.params.get(self.stress_cmds_part) or []
```

Comment:
why do you keep default here?

Reply:
Expected list there. If there is no stress commands in this part, empty list will be at least